RecoBTag/DeepFlavour/interface/c_pf_features_converter.h

```diff
@@ -0,0 +1,263 @@
+#ifndef RecoSV_DeepFlavour_c_pf_features_converter_h
+#define RecoSV_DeepFlavour_c_pf_features_converter_h
+
+#include "RecoBTag/DeepFlavour/interface/deep_helpers.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+
+namespace deep {
+
+  // conversion map from quality flags used in PV association and miniAOD one
+  constexpr int qualityMap[8]  = {1,0,1,1,4,4,5,6};
+
+  enum qualityFlagsShiftsAndMasks {
```

Comment:
Here again it seems better to put the const and enum in the scope of the converter class.